docs/getting-started.md

```diff
@@ -21,6 +21,26 @@ The Client SDK is a set of user-facing programs to interact with Ledger and Audi
 ScalarDL (Ledger and Auditor) abstracts data as a set of assets, where each asset is composed of the history of a record identified by a key called `asset_id` and a historical version number called `age`.
 In this document, you will create a very simple application to manage an asset's status using ScalarDL Client SDK.
 
+## Install a JDK
+
+Because ScalarDL is written in Java, using Java is one of the easist ways to build your ScalarDL applications.
+In such a case, you must have one of the following Java Development Kits (JDKs) installed in your environment:
+
+- [Oracle JDK](https://www.oracle.com/java/technologies/downloads/) LTS version (8, 11, or 17)
+- [OpenJDK](https://openjdk.org/install/) LTS version (8, 11, or 17)
+
+Also, since ScalarDL is built with JDK 8, you must use JDK 8 to build your contracts.
+If you use JDK versions other than 8, specify `-release 8` option for `javac` or define what toolchain to use in your Gradle and Maven configurations.
```

Comment:
Can we highlight this part as Warning or Note in the different color,  just like ScalarDB document, so that the customers don't miss it? 
(https://scalardb.scalar-labs.com/docs/latest/getting-started-with-scalardb/)

Reply:
@NatsukiMiyamoto Yes, we can. But, `Warning` and `Note` are something we use for side information, so I'm not sure if they are proper. 
@josh-wong What would be a good way to emphasize an important description?

Reply:
I think we should use an `Attention` notice in this case.

My suggestion below includes [@/komamitsu suggested change](https://github.com/scalar-labs/scalardl/pull/54/files#r1483902153).

```suggestion
{% capture notice--warning %}
**Attention**

Since ScalarDL is built on JDK 8, you must use JDK 8 to build your contracts.
If you use a version other than JDK 8, specify the `--release 8` option for `javac` or define which toolchain to use in your Gradle and Maven configurations.
{% endcapture %}

<div class="notice--warning">{{ notice--warning | markdownify }}</div>

```

Reply:
I agree with Josh. Attention would be the right phrase for this case. 
Thank you :)